localgov_subsites_extras.module

```diff
@@ -69,5 +71,40 @@ function localgov_subsites_extras_entity_bundle_create($entity_type_id, $bundle)
     $type->setThirdPartySetting('menu_ui', 'available_menus', $availableMenus);
     $type->save();
   }
+}
+
+// phpcs:disable
```

Comment:
Isn't it better to make the code meet Drupal standards?

Reply:
Yeah... This is left over from this module being a prototype! This isn't there anymore now we're using property promotion.

Reply:
Ah, wrong file... Drupal's code style rules don't let you implement hooks for other modules like I've done here, as they insist on the function prefix being the module name. This is just a temporary thing anyway, as I'd like to move those hooks to the modules that provide those types & fields.

Reply:
Just taking a quick look, would an event and event subscriber be better here? And then the module can provide appropriate events for known localgov drupal content types until we can agree on a place to move it.